### PR TITLE
fix(material-experimental/mdc-slider): resolve warnings about active event listeners

### PR DESCRIPTION
**Pull request by crisbeto**
Fixes some warnings in the MDC-based slider that were logged, because we were implicitly binding active event listeners. These changes make them explicit.

### CODE REVIEW

File: src/material-experimental/mdc-slider/slider.ts [hunk 1]
```diff
@@ -232,39 +235,35 @@ export class MatSlider implements AfterViewInit, OnChanges, OnDestroy, ControlVa
         // as they will prevent the default behavior. Additionally we can't run these event
         // handlers outside of the Angular zone because we rely on the events to cause the
         // component tree to be re-checked.
-    this._elementRef.nativeElement.addEventListener(evtType, handler),
+        this._elementRef.nativeElement.addEventListener(evtType, handler, activeListenerOptions),
```
Comment (devversion) [inline src/material-experimental/mdc-slider/slider.ts]: Sounds like something MDC should pass through to foundations IMO. I'm good with this change for now, but the problem here is that there are multiple events registered, but only one needs `passive: false`. Others can be passive.

cc. @mmalerba @jelbourn. I can create a JIRA/ticket for that if we all agree.

Comment (mmalerba) [reply src/material-experimental/mdc-slider/slider.ts]: Yeah we should file an issue to make this part of the adapter's API. @crisbeto can you leave a TODO to clean up once that's added to the API?

Comment (crisbeto) [reply src/material-experimental/mdc-slider/slider.ts]: I've added the TODO.

Comment (devversion) [reply src/material-experimental/mdc-slider/slider.ts]: 👍 @crisbeto do you want to create the issue upstream? or @mmalerba will you talk to them?

Comment (mmalerba) [reply src/material-experimental/mdc-slider/slider.ts]: They do have plans to revise these APIs: https://github.com/material-components/material-components-web/issues/2813

**Pull request merged**